core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -321,6 +321,16 @@ package object config {
       .intConf
       .createWithDefault(3)
 
+  private[spark] val REDUCER_MAX_BLOCKS_IN_FLIGHT_PER_ADDRESS =
+    ConfigBuilder("spark.reducer.maxBlocksInFlightPerAddress")
+      .doc("This configuration limits the number of remote blocks being fetched from a given " +
+        " host port at any given point. When external shuffle is enabled and a large number of " +
```

Comment:
At lease we should state the configuration doesn't necessarily go with external shuffle service.